src/components/heatmap/HeatmapCellColorCanvas.js

```diff
@@ -1,8 +1,9 @@
 import React from 'react';
 
 import { setImageDataRGBA, getImageRendering } from './utils';
+import AbstractHeatmapCanvas from './AbstractHeatmapCanvas';
 
-export default class HeatmapCellColorCanvas extends React.Component {
+export default class HeatmapCellColorCanvas extends AbstractHeatmapCanvas {
```

Comment:
I know this is what I do, but the real React docs favor composition over inheritance. https://reactjs.org/docs/composition-vs-inheritance.html. I don't actually have a clear sense of what problems we're going to run into, but probably better to ween ourselves from this in the long run.